package.json

```diff
@@ -12,6 +12,7 @@
     "@testing-library/user-event": "^7.1.2",
     "axios": "^0.19.2",
     "dotenv": "^8.2.0",
+    "history": "^4.10.1",
```

Comment:
I did need to add a package to be able to test the history object.

Reply:
ok. good to know.